pipeline_steps/influx.groovy

```diff
@@ -0,0 +1,71 @@
+def setup(){
+  instance_name = "INFLUX"
+  pubCloudSlave.getPubCloudSlave(instance_name: instance_name)
+  common.override_inventory()
+  try{
+    common.conditionalStage(
+      stage_name: "Influx",
+      stage:{
+        withCredentials([
+          file(
+            credentialsId: 'id_rsa_cloud10_jenkins_file',
+            variable: 'jenkins_ssh_privkey'
+          ),
+          string(
+            credentialsId: "SSH_IP_ADDRESS_WHITELIST",
+            variable: "SSH_IP_ADDRESS_WHITELIST"
+          ),
+        ]){
+          dir(${env.WORKSPACE}){
+            unstash "pubcloud_inventory"
+          }
+          dir('rpc-gating/playbooks'){
+            git branch: env.RPC_GATING_BRANCH, url: env.RPC_GATING_REPO
```

Comment:
This will checkout the rpc-gating repo into rpc-gating/playbooks. So the playbooks will be at rpc-gating/playbooks/playbooks. Same for maas below.